java/src/test/java/com/saucelabs/simplesauce/SauceOptionsTest.java

```diff
@@ -33,26 +33,28 @@ public void sauceSession_defaultSauceOptions_returnsChromeBrowser() {
 
         sauceSession = new SauceSession(options,fakeRemoteDriver, fakeEnvironmentManager);
         sauceSession.start();
-        String actualBrowser = sauceSession.currentSessionCapabilities.getCapability("browserName").toString();
-        assertEquals("Chrome", actualBrowser);
+        String actualBrowser = sauceSession.getCurrentSessionCapabilities().getCapability("browserName").toString();
+        assertEquals("chrome", actualBrowser);
```

Comment:
@mmerrell were you able to get this to pass because these strings are case sensitive and the `actualBrowser` should come back as `Chrome`?